modules/bullet/rigid_body_bullet.cpp

```diff
@@ -433,10 +444,20 @@ bool RigidBodyBullet::add_collision_object(RigidBodyBullet *p_otherObject, const
 	cd.other_object_shape = p_other_shape_index;
 	cd.local_shape = p_local_shape_index;
 
+	curr_collision_traces->write[collisionsCount] = p_otherObject;
+
 	++collisionsCount;
 	return true;
 }
 
+bool RigidBodyBullet::was_colliding(RigidBodyBullet *p_other_object) {
+	for (int i = prev_collision_count - 1; 0 <= i; --i) {
```

Comment:
It would be more clear to write `i >= 0` instead of `0 <= i`.

Reply:
While I agree, all the code in `modules/bullet/*` uses this style (Andrea's preference I guess), so changing it just here wouldn't make much sense. Since Andrea is the main maintainer of the bullet module, I think we can let him use the style he likes best as long as it's fit clang-format rules.